lib/public/views/Logs/ColumnNames/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const COLUMN_NAMES = {
```

Comment:
Not sure if `COLUMN_NAMES` fits here as it is responsible for a lot more.

Reply:
Renaming it to active columns along with the folder where the file is in.